### PR TITLE
Add Rekono Provider

### PR DESCRIPTION
**Pull request by nunar**

### CODE REVIEW

File: src/Rekono/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+return Socialite::driver('rekono')->redirect();
+```
+
+### Returned User fields
+
+- ``id``
+- ``name``
+- ``email``
+
+More fields are available under the user subkey:
```
Comment (atymic) [inline src/Rekono/README.md:50]: Could we add a link to the docs here?

File: src/Rekono/composer.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "authors": [
+        {
+            "name": "Anze Nunar",
+            "email": "anze@nunar.si"
+        }
+    ],
+    "require": {
+        "php": "^7.2",
+        "ext-json": "*",
+        "socialiteproviders/manager": "~2.0 || ~3.0"
```
Comment (atymic) [inline src/Rekono/composer.json:14]: Can drop v2 here.

File: src/Rekono/Provider.php [hunk 3]
```diff
@@ -0,0 +1,86 @@
+<?php
+
+namespace SocialiteProviders\Rekono;
+
+use SocialiteProviders\Manager\OAuth2\AbstractProvider;
+use SocialiteProviders\Manager\OAuth2\User;
+
+class Provider extends AbstractProvider
```
Comment (atymic) [inline src/Rekono/Provider.php:8]: Can we have an `@see` docblock comment with a link to the docs please :)

File: src/Rekono/composer.json (hunk 2 above)
Comment (nunar) [reply src/Rekono/composer.json]: Done.

File: src/Rekono/README.md (hunk 1 above)
Comment (nunar) [reply src/Rekono/README.md]: A short description of the Rekono provider and instructions for obtaining technical documentation were added.

File: src/Rekono/Provider.php (hunk 3 above)
Comment (nunar) [reply src/Rekono/Provider.php:8]: See [comment above](https://github.com/SocialiteProviders/Providers/pull/547#discussion_r506488504)

**Review by atymic — APPROVED** [review]

**Pull request merged**